setup.py

```diff
@@ -1,46 +1,35 @@
 #!/usr/bin/env python
 # -*- encoding: utf-8 -*-
-from __future__ import absolute_import
+"""Setup routine for OMMBV.
 
-import os
-from setuptools import setup, find_packages
+Note
+----
+package metadata stored in setup.cfg
+
+"""
 
-here = os.path.abspath(os.path.dirname(__file__))
-version_filename = os.path.join('OMMBV', 'version.txt')
-with open(os.path.join(here, version_filename)) as version_file:
-    version = version_file.read().strip()
+import os
+from setuptools import setup
 
 # Include extensions only when not on readthedocs.org
 if os.environ.get('READTHEDOCS', None) == 'True':
     extensions = []
```

Comment:
```suggestion
    from setuptools import setup
    extensions = []
```

Reply:
I've found the first setup needs to be at top level for `setup.py develop` to work.